docs/rules/padding-line-between-statements.md

```diff
@@ -0,0 +1,229 @@
+# Require or disallow padding lines between statements (padding-line-between-statements)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+This rule requires or disallows blank lines between the given 2 kinds of statements.
+Properly blank lines help developers to understand the code.
+
+For example, the following configuration requires a blank line between a variable declaration and a `return` statement.
+
+```js
+/*eslint padding-line-between-statements: ["error", [
+    { blankline: "always", prev: "var", next: "return" }
+]]*/
+
+function foo() {
+    var a = 1;
+
+    return a;
+}
+```
+
+## Rule Details
+
+This rule does nothing if no configuration.
+
+A configuration is an array which has 3 elements; a linebreak type and 2 kind of statements. For example, `{ blankline: "always", prev: "var", next: "return" }` is meaning "it requires one or more blank lines between a variable declaration and a `return` statement."
+You can supply any number of the configuration. If an statement pair matches some configurations, it chooses the last matched configuration.
```

Comment:
Nitpick: I think it would be better to word this as:

> You can supply any number of configurations. If an statement pair matches multiple configurations, the last matched configuration will be used.